ui/components/component-library/banner-base/banner-base.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jest/require-top-level-describe */
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { renderWithUserEvent } from '../../../../test/lib/render-helpers';
+
+import { Icon, ICON_NAMES } from '..';
+import { BannerBase } from './banner-base';
+
+describe('BannerBase', () => {
+  it('should render bannerbase element correctly', () => {
+    const { getByTestId, container } = render(
+      <BannerBase data-testid="banner-base" title="Bannerbase test">
+        should render bannerbase element correctly
+      </BannerBase>,
+    );
+    expect(getByTestId('banner-base')).toHaveClass('mm-banner-base');
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render with added classname', () => {
+    const { getByTestId } = render(
+      <BannerBase
+        className="mm-banner-base--test"
+        data-testid="banner-base"
+        title="Bannerbase test"
+      >
+        should render bannerbase element correctly
+      </BannerBase>,
+    );
+    expect(getByTestId('banner-base')).toHaveClass('mm-banner-base--test');
+  });
+
+  it('should render bannerbase title', () => {
+    const { getByText } = render(<BannerBase title="Bannerbase title test" />);
+    expect(getByText('Bannerbase title test')).toHaveClass(
+      'mm-banner-base__title',
+    );
+  });
+
+  it('should render bannerbase description', () => {
+    const { getByText } = render(
+      <BannerBase>Bannerbase description test</BannerBase>,
+    );
+    expect(getByText('Bannerbase description test')).toHaveClass(
+      'mm-banner-base__description',
+    );
+  });
+
+  it('should render bannerbase action button', () => {
+    const { getByTestId } = render(
+      <BannerBase
+        title="Action prop demo"
+        actionButtonLabel="Action"
+        actionButtonProps={{
+          icon: ICON_NAMES.ARROW_2_RIGHT, // TODO: change to iconName
```

Comment:
Can this be updated now?
```suggestion
          iconName: ICON_NAMES.ARROW_2_RIGHT,
```